tasks/omero-install.yml

```diff
@@ -108,11 +122,57 @@
     state: present
     virtualenv: "{{ omero_server_basedir }}/venv"
     virtualenv_site_packages: true
-  when: omero_server_virtualenv
+  when: omero_server_virtualenv and not omero_server_python3
+  notify:
+    - omero-server rewrite omero-server configuration
+    - omero-server restart omero-server
+
+# TODO: figure out dependencies
+- name: omero server | setup virtualenv3
+  become: true
+  pip:
+    name: "{{ omero_server_python_requirements + omero_server_python_addons }}"
+    state: present
+    virtualenv: "{{ omero_server_virtualenv_basedir }}"
+    virtualenv_command: /usr/bin/python3 -mvenv
+  when: omero_server_python3
   notify:
     - omero-server rewrite omero-server configuration
     - omero-server restart omero-server
 
+# TODO: Remove when https://github.com/ome/omego/pull/126 is released
+- name: omero server | patch omego/external.py
```

Comment:
Can you try consuming https://github.com/ome/ansible-role-omego/pull/6 which includes these patches?

Reply:
Updated to use omego 0.7.0. Note this role doesn't use the separate omego role for python 3, instead it is installaed into the server venv. omero-web is pip installable so nothin else requires omego.

Reply:
> this role doesn't use the separate omego role for python 3

This broke the docker certainly. Not sure how wide ranging that will be.

Reply:
Does this mean you are proposing to deprecate https://github.com/ome/ansible-role-omego in the mid-term and have `omego` handled as an internal task wherever needed? I don't have any conceptual objection to this strategy as long as we collectively agree on it and discuss all the  implications like the Docker case.

If we decided to deprecate `ome.omego`, an additional thought is that this role could unconditionally creating a virtualenv and install `omego`. The top-level flag would determine whether the virtualenv is also used for the OMERO dependencies or only for deployment.

Reply:
If you think it's useful we can keep `ome.omego` as a standalone role, but we'll need to handle python2 vs python3 in that role. I don't think we should have a flag in this role to switch between them, it increases the testing matrix for little benefit.